### PR TITLE
Error handling

### PR DESCRIPTION
**Pull request by Neko250**
- `add`: error handling middleware
- `change`: minor refactors

### CODE REVIEW

File: api/_bot/index.js [hunk 1]
```diff
@@ -1,18 +1,25 @@
 import Telegraf from 'telegraf'
-import { debug, replyMessage } from './middlewares'
 import { help, start } from './commands'
+import { debug, error, replyTo } from './middleware'
```
Comment (melchor629) [inline api/_bot/index.js]: ??

File: api/_bot/middleware/reply-to.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 export default (ctx, next) => {
-  ctx.replyMessage = (msg, opts = {}) =>
```
Comment (melchor629) [inline api/_bot/middleware/reply-to.js]: this does not reply to a user in a sense of `@user <message>` replies to the message itself so it does not make sense to rename it

File: api/_bot/index.js (hunk 1 above)
Comment (Neko250) [reply api/_bot/index.js]: reverted

File: api/_bot/middleware/reply-to.js (hunk 2 above)
Comment (Neko250) [reply api/_bot/middleware/reply-to.js]: reverted

**Pull request merged**